src/js/util/AuthService.jsx

```diff
@@ -26,17 +28,21 @@ class AuthService extends EventEmitter {
   }
 
   _doAuthentication(authResult){
-    // Saves the user token
-    this.setToken(authResult.idToken)
-    // Async loads the user profile data
-    this.lock.getProfile(authResult.idToken, (error, profile) => {
-      if (error) {
-        console.log('Error loading the Profile', error)
-      } else {
-        this.setProfile(profile);
-        this._createUser();
-      }
-    })
+    if (authResult.state.includes('linking')) {
```

Comment:
Attempting to run on this branch as master wasn't working for me. 
I get an error here: "Cannot read property 'includes' of undefined"
Where is authResult.state defined?